.github/workflows/ci.yml

```diff
@@ -42,8 +42,10 @@ jobs:
         run: |
           set -o pipefail; mvn clean install -Pgenerate-docs | tee ./mvn.log; set +o pipefail
           if [[ $(grep -c "overlapping classes" ./mvn.log) -gt 0 ]];then
```

Comment:
Maybe we could count the overlapping classes like following.
```
cat ./mvn.log | grep -E -v 'flink-runtime-.*.jar, flink-kubernetes-operator-.*.jar define 2 overlapping classes' | grep -c "overlapping classes" -
```

Reply:
I like this